### ISSUE TITLE
babel-plugin-tester@canary

### ISSUE THREAD

**Issue by Xunnamius** [babel-utils/babel-plugin-tester#267]
This is more of a reminder so I can keep track of this:

[`babel-plugin-tester@canary`](https://www.npmjs.com/package/babel-plugin-tester/v/12.0.0-canary.1) has been out for a while now, but it is blocked until the release of [`jest@30`](https://github.com/jestjs/jest/milestone/15). This is because, even though BPT doesn't technically rely on `jest@30`, `jest@<30` does not work with `prettier@>=3` yet `babel-plugin-tester@canary` cannot work with both `prettier@>=3`'s sync interface and `prettier@<3`'s async interface simultaneously without increased complexity of diminishing value.

tl;dr `babel-plugin-tester@canary` works with the latest/prerelease versions of Prettier and Jest, so if you're using `prettier@>=3` and/or `jest@>=30`, you may also want to use `babel-plugin-tester@canary`.
